js/content/store.js

```diff
@@ -2662,53 +2660,41 @@ let FundsPageClass = (function(){
             );
         }
 
-        let currency = await Price.parseFromString(price, Currency.storeCurrency);
+        let currency = CurrencyRegistry.fromType(Currency.storeCurrency);
+        let minValue = currency.valueOf(price);
+        let symbol = currency.format.symbol;
+        let step = currency.format.decimalPlaces === 0 ? 1 : .01;
```

Comment:
This ensures that it's using the smallest step possible, although I doubt that anyone will ever use that
```suggestion
        let step = Math.pow(10, -currency.format.decimalPlaces);
```